src/Tribe/Rewrite.php

```diff
@@ -676,4 +697,46 @@ protected function get_option_controlled_slug_entry( array $localized_matchers,
 
 		return $entry;
 	}
+
+	/**
+	 * Returns the map of localized bases for the specified text domains.
+	 *
+	 * The bases are the ones used to build the permalinks, the domains are those of the currently activated plugins
+	 * that include a localized rewrite component.
+	 *
+	 * @since TBD
+	 *
+	 * @param array<string> $bases   The bases to set up the locale translation for.
+	 * @param array<string> $domains A list of text domains belonging to the plugins currently active that handle and
+	 *                               provide support for a localized rewrite component.
+	 *
+	 * @return array<string,string> A map relating the bases in their English, lowercase form to their current locale
+	 *                              translated form.
+	 */
+	public function get_localized_bases( array $bases, array $domains ) {
+		$locale             = get_locale();
+		$cache_key          = __METHOD__ . md5( json_encode( array_merge( $bases, $domains, [ $locale ] ) ) );
```

Comment:
:warning: json_encode() is discouraged. Use wp_json_encode() instead.

<sub>hash: afc2d49c6533e77638275ca8b40f4169</sub>